<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_supplier</class>
 <widget class="QDialog" name="add_supplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm nhà cung cấp</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thêm nhà cung cấp</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tên nhà cung cấp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Địa chỉ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_sdtNCC">
        <property name="text">
         <string>Số điện thoại</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_emailNCC">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_tenNCC"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_dia_chi"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_sdtNCC"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_emailNCC"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tv_nha_cung_cap">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
